lib/pbench/test/unit/server/test_server_audit.py

```diff
@@ -58,52 +76,86 @@ def make_audits(self, client, create_user) -> str:
             root = Audit.create(
                 operation=OperationCode.UPDATE,
                 name="second",
+                dataset=drb,
                 user_name="fake",
                 status=AuditStatus.BEGIN,
             )
             f.tick(delta=datetime.timedelta(seconds=5))
-            Audit.create(root=root, status=AuditStatus.FAILURE)
-        return user_id
-
-    def test_get_bad_keys(self, query_get):
-        response = query_get({"xyzzy": "foo"}, HTTPStatus.BAD_REQUEST)
-        assert response.json == {"message": "Unknown URL query keys: xyzzy"}
-
-    def test_get_all(self, query_get, make_audits):
-        """With no query parameters, we should get all audit records"""
-        expected_user_id = make_audits
+            Audit.create(
+                root=root, status=AuditStatus.FAILURE, reason=AuditReason.PERMISSION
+            )
         response = query_get(expected_status=HTTPStatus.OK)
         audits = response.json
-        assert len(audits) == 4
         assert audits[0]["status"] == "BEGIN"
         assert audits[0]["operation"] == "CREATE"
         assert audits[0]["name"] == "first"
-        assert audits[0]["user_id"] == expected_user_id
+        assert audits[0]["user_id"] == str(create_user.id)
         assert audits[0]["user_name"] == "test"
+        assert audits[0]["object_id"] is None
+        assert audits[0]["object_name"] is None
+        assert audits[0]["object_type"] is None
+        assert audits[0]["user_name"] == "test"
+        assert audits[0]["reason"] is None
         assert audits[1]["status"] == "SUCCESS"
         assert audits[1]["operation"] == "CREATE"
         assert audits[1]["name"] == "first"
-        assert audits[1]["user_id"] == expected_user_id
+        assert audits[1]["user_id"] == str(create_user.id)
         assert audits[1]["user_name"] == "test"
+        assert audits[1]["object_id"] is None
+        assert audits[1]["object_name"] is None
+        assert audits[1]["object_type"] is None
+        assert audits[1]["reason"] is None
```

Comment:
Should we be checking `"user_name"` here?  (Ditto for entries 2 and 3?)

Reply:
I think perhaps it is the addition of line 97 which is the problem.  (Does it add anything over line 93?)